README.md

```diff
@@ -283,6 +298,8 @@ We release the components of Knative every 6 weeks. All of these components must
 be moved to the latest "release" of all shared dependencies prior to each
 release.
 
+The docs are released after serving and eventing are released, see [Release docs](#release-docs)
+
```

Comment:
See my comment above, I would suggest making this more prominent and just say: The docs should be released after the code has been released (so that the docs release team does not need to be aware of the details on which repo it depends. E.g. when including examples from r links to the released client tag or client plugins tag, then you would otherwise need to update this instruction here, which can be easily forgotten, too).

Also, as soon as you add a link to released client plugins, you would have to wait anyway until everything is released, as those plugins are the last to be released.